### DISCUSSION TITLE
QoL Mods

### DISCUSSION THREAD

**Discussion by IcedDog** [DM-Earth/Cabricality#52] [Ideas]
so i go through the whole list of not-so-important mods that CAB uses, and here're the mods that are available in fabric but isnt added yet

- [x] [Backslot](https://www.curseforge.com/minecraft/mc-mods/backslot)
- [x] [Bad Wither No Cookie - Reloaded](https://www.curseforge.com/minecraft/mc-mods/bad-wither-no-cookie-reloaded-fabric)
- [ ] [Building wands](https://modrinth.com/mod/building-wands)
- [ ] [Clumps](https://www.curseforge.com/minecraft/mc-mods/clumps)
- [x] [Continuity](https://modrinth.com/mod/continuity)
- [ ] [Decorative Blocks](https://www.curseforge.com/minecraft/mc-mods/decorative-blocks)
- [ ] [Drink Beer](https://www.curseforge.com/minecraft/mc-mods/drink-beer-fabric)
- [ ] [Fabricated Chisel](https://modrinth.com/mod/fabric-chisel/)
- [x] [Leaf Decay](https://www.curseforge.com/minecraft/mc-mods/leaf-decay)
- [x] [lootr](https://www.curseforge.com/minecraft/mc-mods/lootr-fabric)
- [ ] [Macaw's Lights and Lamps](https://www.curseforge.com/minecraft/mc-mods/macaws-lights-and-lamps)
- [x] [Spark](https://www.curseforge.com/minecraft/mc-mods/spark)
- [ ] [Supplementaries](https://www.curseforge.com/minecraft/mc-mods/supplementaries) *1.19.2 only for Fabric*
- [ ] [TheMightyArchitect](https://github.com/LCLPYT/TheFabricArchitect)

---

mods that are not crucial and not origin-based but i personally would like the pack more if added:

- [ ] [Another furniture](https://www.curseforge.com/minecraft/mc-mods/another-furniture)
- [x] [Architecture Extensions](https://modrinth.com/mod/arch-ex/)
- [ ] [Chalk](https://www.curseforge.com/minecraft/mc-mods/chalk-fabric)
- [ ] [Dazzle 2](https://www.curseforge.com/minecraft/mc-mods/dazzle-2)
- [ ] [NiceMod](https://www.curseforge.com/minecraft/mc-mods/nicemod-new-blocks)
- [ ] [Plushie Mod](https://modrinth.com/mod/plushie)
- [ ] [Red Bits](https://www.curseforge.com/minecraft/mc-mods/red-bits)
- [ ] [Redstone Pen](https://www.curseforge.com/minecraft/mc-mods/redstone-pen) (these are to make up for project red)
- [ ] [Tips](https://www.curseforge.com/minecraft/mc-mods/tips)
- [ ] [Urns](https://www.curseforge.com/minecraft/mc-mods/urns) (urgent shortage of small decorative containers like this)
- [ ] [Villager Hats](https://modrinth.com/mod/villager-hats/)
- [ ] [Wild Firefly](https://modrinth.com/mod/wildfirefly)
- [ ] [Windchimes](https://modrinth.com/mod/windchimes/)
- [ ] [Wired Redstone](https://www.curseforge.com/minecraft/mc-mods/wired-redstone)
- [ ] [Workings](https://www.curseforge.com/minecraft/mc-mods/workings)
- [ ] [Custom Portals](https://www.curseforge.com/minecraft/mc-mods/custom-portals)

**Comment by JieningYu** [comment]
Additional Additions is useless in this modpack with tinker's, I think.

**Comment by KrLite** [comment]
Continuity is now in the pack with some brand new connected textures.

**Comment by IcedDog** [comment]
good good

**Comment by JieningYu** [comment]
https://github.com/mrsterner/Supplementaries-Quilt

**Comment by IcedDog** [comment]
its gone 😭

**Comment by JieningYu** [comment]
what

**Comment by JieningYu** [comment]
O the supplementaries quilt

**Comment by IcedDog** [comment]
3 out of 52 task, good luck guys :)
i can help create some custom structures in the mean time 🥇

**Comment by JieningYu** [comment]
supplementaries will be added in 1.19.2 version.

**Comment by JieningYu** [comment]
some mods has been deleted in this list as they won't be added.

**Comment by IcedDog** [comment]
good
